SpatialGDK/Source/SpatialGDK/Public/Interop/CrossServerRPCHandler.h

```diff
@@ -13,7 +13,7 @@ class SpatialEventTracer;
 DECLARE_DELEGATE_RetVal_OneParam(bool, FProcessCrossServerRPC, const FCrossServerRPCParams&);
 DECLARE_DELEGATE_RetVal_OneParam(FCrossServerRPCParams, FTryRetrieveCrossServerRPCParams, const Worker_CommandRequestOp&);
 
-static double CrossServerRPCGuidTimeout = 5.f;
+static double CrossServerRPCGuidTimeout = 30.f;
```

Comment:
Driveby - why is there a static double at in a header? I presume this is meant to be a constant?

Reply:
hm, good point
can make it a constexpr. Would you like to move it as well? Could also see this being an advanced SpatialGDK setting.

Reply:
We can do if we think it's something users need to change but otherwise I think it should be a static const field on teh rpc handler (rather than global).

Also why 30 seconds?

Reply:
I have found 5 to be within the realms of unreliability and random stalls. This is just a finger-in-the-air number, but I thought 30.0 would be better. Can vote/discuss.

Reply:
(as in, I got a 5s stall while testing this that caused issues)

Reply:
Moved into cpp and made constexpr